.travis.yml

```diff
@@ -3,7 +3,12 @@ node_js:
   - "10"
```

Comment:
There's an issue in the latest version of mock-fs that makes this not work in recenter versions of node 10.  https://github.com/tschaub/mock-fs/issues/238

You should probably pin this version at 10.4.1 for now, and create a follow-up issue to update once that issue is resolved.

Reply:
Updated mock-fs to 4.7.0